src/utils.js

```diff
@@ -0,0 +1,9 @@
+export function isBase64Encoded(str) {
+    // Base64 encoded strings can only contain characters from [A-Za-z0-9+/=]
+    const base64RegExp = /^[A-Za-z0-9+/=]+$/;
+    return base64RegExp.test(str);
+}
+
+export function isEmptyObject(obj) {
```

Comment:
Instead of implementing this sort of function, you can use _isEmpty_ function from lodash.
https://lodash.com/docs/4.17.15#isEmpty

Reply:
I checked. Unfortunately this doesn't work as expected. We can stay at this stage on current solution. Maybe we can think about improvement later when we have more time.